hudi-hadoop-common/src/main/java/org/apache/hudi/common/util/HFileUtils.java

```diff
@@ -232,6 +232,15 @@ public byte[] serializeRecordsToLogBlock(HoodieStorage storage,
     return baos.toByteArray();
   }
 
+  /**
+   * Print the record in json format
+   */
+  private void printRecord(String msg, byte[] bs, Schema schema) throws IOException {
+    GenericRecord record = HoodieAvroUtils.bytesToAvro(bs, schema);
+    byte[] json = HoodieAvroUtils.avroToJson(record, true);
+    LOG.error(String.format("%s: %s", msg, new String(json)));
```

Comment:
Just want to make sure this is just metadata and not actual data since that can leak sensitive information into the logs.

If so, also update this to use `{}`